.scripts/prepare-repo.sh

```diff
@@ -27,5 +27,7 @@ rm -rf ${PACK_DIRECTORY}
 
 ls -la .
 
-echo "export PACK_NAME=$PACK_NAME" >> ~/.circlerc
-echo "export FORCE_CHECK_ALL_FILES=true" >> ~/.circlerc
+if [[ "${CIRCLECI}" == "true" ]]; then
```

Comment:
Ah. True. I left CircleCI intact everywhere else, so I did that here too. But, that's not needed since this is the only repo that uses this. I'll just delete this block.

Reply:
dropped this block